crates/fuel-core/src/combined_database.rs

```diff
@@ -108,11 +142,14 @@ impl CombinedDatabase {
     /// A test-only temporary rocksdb database with given rewind policy.
     #[cfg(feature = "rocksdb")]
     pub fn temp_database_with_state_rewind_policy(
```

Comment:
We could use database config here as well

Reply:
Done